komodo/build.py

```diff
@@ -248,8 +248,6 @@ def resolve(input_str):
             msg = "pypi_package_name is only valid when building with pip"
```

Comment:
Why not also remove this? This is guarding the statement you are trying to remove, no?

Reply:
E.g.
```
if "pypi_package_name" in current and make != "pip":
    msg = "pypi_package_name is only valid when building with pip"
    raise ValueError(msg)
```

Reply:
I kept it in case the user would need a hint to how a correct configuration file should look like. But this is probably the wrong place for that error messsage.

Reply:
No, this is probably the place to put this error.